physics/module_mp_thompson.F90

```diff
@@ -92,7 +92,8 @@ MODULE module_mp_thompson
 !.. scheme.  In 2-moment cloud water, Nt_c represents a maximum of
 !.. droplet concentration and nu_c is also variable depending on local
 !.. droplet number concentration.
-      REAL, PARAMETER :: Nt_c = 100.E6
+      REAL, PARAMETER :: Nt_c_o = 50.E6
```

Comment:
I wish to request the original line of code remain as it was - even as a comment line.  I typically do not request useless comment lines remain, but note how the code changes do not even mention the new nomenclature means "o" for ocean or "l" for land.  For continuity, I believe the old line as a comment is acceptable in this case.

Reply:
It is fine with me.

Reply:
Modified as suggested by gthompsnWRF